apps/daimo-mobile/src/view/screen/HomeScreen.tsx

```diff
@@ -194,7 +206,9 @@ function HomeScreenInner({ account }: { account: Account }) {
                 account.suggestedActions.length > 0 &&
                 netState.status !== "offline" &&
                 isActionVisible
-              ) && <Spacer h={64} />}
+              ) && <Spacer h={allComplete ? 64 : 28} />}
+              {!allComplete && <CompleteOnboarding />}
```

Comment:
i don't think we can fit *both* a suggested action and the onboarding checklist, right?

if `!allComplete`, i think we can hide the suggested action & show onboarding checklist instead